imports/plugins/included/marketplace/lib/api/marketplace.js

```diff
@@ -0,0 +1,50 @@
+import { Meteor } from "meteor/meteor";
+import { Packages } from "/lib/collections";
+import { Reaction } from "/lib/api";
+
+const name = "reaction-marketplace";
+
+/*
+No need for this anymore
+Use Reaction.getPackageSettings(name) instead
+export function getPackageSettings() {
+
+  return Reaction.getPackageSettings(name)
+}*/
+
+/*
+ No need for this anymore
+ Use Reaction.isPackageEnabled(name) instead
+export function isEnabled() {
+  const packageInfo = getPackageSettings();
+
+  if (typeof packageInfo.enabled === "boolean") {
+    return packageInfo.enabled;
+  }
+
+  return false;
+}*/
+
+export function hasMarketplaceGuestAccess() {
+  const currentUser = Meteor.user();
+
+  // parent shop owners have full access
+  if(Roles.getGroupsForUser(currentUser, ['owner']).length) {
+    return true;
+  }
+
+  const packageSettings = Reaction.getPackageSettings('reaction-marketplace');
```

Comment:
`getPackageSettings` is a much cleaner implementation . We should create an issue to clean up the implementation in the rest of the core packages, and make sure that docs are updated with these new methods.

A comment on styles,  `eslint` should be warning on these,  but http://eslint.org/docs/rules/no-extra-semi and http://eslint.org/docs/rules/quotes are common offenders in this code.  `eslint -fix` is the easiest way to clean all those up.   Generally, we've been "double quote" advocates.